packages/server/src/cli.ts

```diff
@@ -218,14 +223,7 @@ const bold = (text: string | number): string | number => {
 					return forkModule(options.env.AMD_ENTRYPOINT, args, options, dataDir);
 				}
 
-				if (isCli) {
-					return spawn(process.execPath, [path.join(buildDir, "out", "cli.js"), "--fork", modulePath, "--extra-args", JSON.stringify(args), "--data-dir", dataDir], {
```

Comment:
I wasn't sure when this gets called, so I haven't been able to test it, but a regular fork should work right?

Reply:
I believe so